FinniversKit/Sources/Components/HTMLLabel/HTMLLabel.swift

```diff
@@ -0,0 +1,66 @@
+public class HTMLLabel: Label {
+    private var htmlText: String?
+    private var styleMap: HTMLStyler.StyleMap
+    private var additionalStyleMap: HTMLStyler.StyleMap?
+
+    public override var textColor: UIColor! {
+        didSet {
+            styleMap[.textColor] = textColor
+        }
+    }
+
+    public init(style: Style, styleMap: HTMLStyler.StyleMap = [:], withAutoLayout: Bool = false) {
+        self.styleMap = styleMap
+        super.init(style: style, withAutoLayout: withAutoLayout)
+    }
```

Comment:
You set `numberOfLines` to `0` everywhere this view is used. Should it maybe just be set here for this view?

Reply:
Not sure about it 🤔 It's still a `Label` and we don't set it there either...

Reply:
That's true. But now that we have this init, how about adding it as an param here instead with a default value of either 0 or 1, then?

```swift
public init(
    style: Style,
    styleMap: HTMLStyler.StyleMap = [:],
    numberOfLines: Int = 0,
    withAutoLayout: Bool = false
) {
    self.styleMap = styleMap
    super.init(style: style, withAutoLayout: withAutoLayout)
    self.numberOfLines = numberOfLines
}
```